tensorflow/core/kernels/cwise_ops.h

```diff
@@ -153,6 +153,27 @@ struct functor_traits<safe_div_or_mod_op<T, DivOrMod>> {
   };
 };
 
+template <typename T>
+struct unsafe_div_op {
+  EIGEN_EMPTY_STRUCT_CTOR(unsafe_div_op)
+  EIGEN_DEVICE_FUNC EIGEN_STRONG_INLINE const T operator()(const T& a,
+                                                           const T& b) const {
+    if (TF_PREDICT_TRUE(b != 0)) {
```

Comment:
I would just avoid `TF_PREDICT_TRUE` here. It places the rest of the code in the cold section, which will pretty much always generate a page fault, so for this somewhat infrequent path, it will be very slow. I feel that it's better to take a little of code bloat here than taking a huge hit for some users.